source/features/index.tsx

```diff
@@ -85,11 +93,20 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 	}
 
 	if (document.body.classList.contains('logged-out')) {
-		console.warn('%cRefined GitHub%c is only expected to work when you’re logged in to GitHub.', 'font-weight: bold', '');
+		console.warn('Refined GitHub is only expected to work when you’re logged in to GitHub. Errors will not be shown.');
+		features.error = () => {/* No logging */};
+		logError = () => {/* No logging */};
 	}
 
 	if (select.exists('html.refined-github')) {
-		console.warn('Refined GitHub has been loaded twice. If you didn’t install the developer version, this may be a bug. Please report it to: https://github.com/sindresorhus/refined-github/issues/565');
+		console.warn(stripIndent(`
+			Refined GitHub has been loaded twice. This may be because:
+
+			• You loaded the developer version, or
+			• The extension just updated
+
+			If you see this at every load, please open an issue mentioning the browser you're using and the URL where this appears.
+		`));
```

Comment:
## Improved copy for https://github.com/sindresorhus/refined-github/issues/565

That issue is outdated and people shouldn't refer to it. A new issue is better.